website/apps/transcripts/views.py

```diff
@@ -395,7 +395,7 @@ def get_context_data(self, page, transcript=None):
 
     def first_log_line(self):
         try:
-            return next(self.log_lines())
+            return next(iter(self.log_lines()))
```

Comment:
I suspect we actually want `log_lines()` to return an iterator, but that may break other things that treat it as a list.

Reply:
Each class in this file actually has their own implementation of `log_lines()`, so it could go either way. This method is actually only used once, in `first_log_line()`, directly above. Given that, I've removed `OriginalView.log_lines()` to make this clearer and simpler, and in the process spotted some code in `backend.api.BaseQuery.__iter__()` where we can clarify it using `yield from` instead of making an iterator by wrapping a list that wraps a generator.